umbraco/Umbraco-CMS
# V13: Delivery API composite id handler

# Notes
- Implement `IDeliveryApiCompositeIdHandler`
- Refactor `DeliveryApiContentIndex` & `DeliveryApiContentIndexHandleContentTypeChanges` to use `IDeliveryApiCompositeIdHandler`.

# How to test
- Add a language so you have atleast 2 languages
- Create a document type with `allow vary by culture` set to true.
- Create a document in both languages.
- Go to settings -> Examine management -> `DeliveryApiContentIndex`.
- Search for the document you created
- You should see the same `__NodeId` as in v13/dev:

![image](https://github.com/umbraco/Umbraco-CMS/assets/70372949/147217d9-870e-414e-a668-d00b31e44f15)
